### ISSUE TITLE
[DISCUSSION] Who can report what?

### ISSUE THREAD

**Issue by dshafik** [derickr/php-code-of-conduct#24]
After a conversation on IRC, one of the primary concerns is about _who_ can report bad behavior — that is, can anyone report bad behavior, even if not directed at them.

This gets further complicated when a person does something that doesn't not offend the receiver of the action, but the action itself is found offensive by someone else. For example: if someone mis-genders someone, and they don't care, but someone else find that quite abhorrent.

I'd like to propose the following:
- Anyone can report bad behavior however:
  - If they weren't the direct recipient of that behavior, we must first reach out to the recipient and ask if they would like to pursue the matter
  - If the person falls under the second paragraph above, it should go no further than a friendly warning like "Hey, some people dislike that you misgendered somebody, can you please be more mindful of that in future? Thanks! :)" (though judgement, as always, should be used, if it's persistent for example.)

This will hopefully limit the damage that can be done by people wanting to pursue vendettas and other trollish behavior under the protection of the CoC.

**Comment by Swader** [comment]
Can you give some examples of such behavior?

**Comment by dshafik** [comment]
@swader I already gave one in the description; could you be more specific?

**Comment by Swader** [comment]
What does it mean to "mis-gender someone", this sounds too vague to me? Perhaps a specific example of an exchange would be helpful.

**Comment by dshafik** [comment]
@Swader: To mis-gender someone means to use the incorrect pronouns for that person, as can happen often to people with gender neutral names like Chris, and Sam.

Example exchange:

Lee> Chris, You are obviously a smart _guy_, but have you considered X?
Chris> I've considered both X and even Y, and neither is an issue as you can see from this source: LINK. Also, I'm a girl, not a guy.
Lee> oops, sorry about that!

So, in this exchange, Lee made a mistake, and Chris probably realizes that and doesn't care. She set the record straight and that should be that.

However perhaps Fran (another gender neutral name in its short form I'll note) is worried that Chris was actually offended by this and reports it.

What should happen? Well, perhaps Chris really was upset by it but too afraid to bring it up, so the CoC team would reach out and check. And follow up appropriately.

In all likelihood that contact would result in a "nope, totally fine, not a big deal, honest mistake" and that's that.

Perhaps though, the act itself offended Fran, and she complains on her own behalf. Being as she was not the direct "victim", nothing should really happen. If it's a common thing for the person (especially common for some cultures, like Brazil, I've noticed) then a gentle comment to that person to be more aware in future might be necessary but nothing more.

HOWEVER, just to bring this full circle, as I'm sure you think this example is ALWAYS trivial, and stupid. Well, it's not. 

It is a common bully tactic to intentionally mis-gender trans and non-binary people, as well as call them by their original gendered names rather than their chosen names. It is often done by people who don't believe that being trans or gender fluid or non-binary are a real thing and refuse to respect the persons wishes to protest against it.

 This WOULD be a behavior that should be disciplined. (Though of course we are all human and a slip up here and there is fine, this is more about persistence.)

Does that help?

**Comment by Swader** [comment]
It does, thank you.

So, does constantly having to think about removing gender non-neutral pronouns from every uttered sentence not equate to walking on eggshells? To me, that's just too much energy for something so trivial.

But aside from my own _feelings_ on the matter, please note that we in the anti-CoC camp are mostly worried about the Frans of the world scouring the internet and just using every single opportunity to complain about someone. Rest assured, these people exist in great numbers, and the CoC gives them the power to keep complaining about someone until they find that one little tidbit that actually _does_ offend random individual xyz and kick up a fuss around their private life. That is what worries us - the CoC giving power to the bad apples, while adding virtually none to the good ones - it's not like **I** will be entitled to protection from the committee if someone just targets me for whichever reason, stalks me, and uses every available opportunity to try and smear me in front of the people who deliberate on these things. 

Even if I knew they were doing it (which I won't since it's non-public to report people), I couldn't do anything about it because I'm not a "minority" in spite of having achieved everything in life on my own with no help from family or country, and despite having been brought up in dirt poor conditions of a near third world country where I was bullied and excluded all my life because I was an actual minority (atheist in a hyper religious and superhostile environment), and rising above it on merit alone.

Basically, we who have actually been targeted minorities all our lives are now considered "the privileged majority" simply because we _worked our way into it_ - in fact, simply because of _looking like we belong_. That is true discrimination - because I'm a white dude does not mean it's privileged, because I sure as hell wasn't when I was growing up and building my career. I wasn't born into my position, and I sure as hell wasn't given it willy nilly. I worked for years to get to where I am.

**Comment by dshafik** [comment]
@Swader 

> So, does constantly having to think about removing gender non-neutral pronouns from every uttered sentence not equate to walking on eggshells? To me, that's just too much energy for something so trivial.

While it might seem trivial to you, it's not to some people who have struggled to get their gender accepted. And this is just a very specific example we shouldn't follow to far down the rabbit hole. Id like to point out that excepting the fictitious exchange above, we haven't used a single pronoun, never mind a gendered one. It's not that big a deal to avoid and simply use they, or other non-gendered variants (I like "folks" for groups instead of "guys", "y'all" is also handy as my default after living in Florida for 13 years :P)

> Rest assured these people exist in great numbers

Compared to what? Anti-CoCers? A majority of all people? Have you seen 4chan for example? A relatively small number of people in the grand scheme of humanity, yet a lot of people being trolls and such… 

Again, as I said in Twitter, if you trust our judgement without a CoC, then trust that same judgement to enforce it. Furthermore, the repeat BS reporters will be noticed but those handling it, and if they do somehow stumble upon a genuine grievance, then GOOD! Bear in mind it will be the team, not the accuser reaching out to the "victim". Are you saying you don't WANT someone who has been harassed and is too afraid to come forward to get help handling it??

> it's not like I will be entitled to protection from the committee if someone just targets me for whichever reason, stalks me, and uses every available opportunity to try and smear me in front of the people who deliberate on these things.

Whatever gave you the impression a CoC was _only_ to protect minorities? It applies equally to everyone and IMO anyone doing that to you should be disciplined for it, with as much severity as any one doing it to anyone else, minority or not. Also, the team handling it will be able to spot the repeated reports against you and if you feel you might be being unfairly targeted then ask them to look into it.

**Comment by leedavis81** [comment]
I have to say I can completely understand the behaviours and situation Bruno (@Swader) is trying to avoid being tangled up in here.

@dshafik in your example it could be the case (and I can say I've witnessed this first hand) that Chris either straight-off-the-bat or after intervention from Fran begins a campaign of assault on Lee. He gets labelled an entitled bigot, part of the patriarchy, needs to check his white privilege etc etc. It all get's out of hand and a simple apology no longer suffices.

Outrage ensues, from both sides. Social justice warriors are all over it, all the rest are shouting "PC gone mad" and it all perpetuates into big messy {insert name}gate.

The fact is, people are always going to take it too far. This is the internet, and trying to police it is simply futile. EVERY movement (unfortunately) will have their extremists. Where you set the bar will differ from person to person, but have no doubts, no matter how thick skinned you are there will be someone out there that will cross it and set your blood boiling. The rationally inept (or just plain bullies).

Anyway, when this does occur, it's important to consider (on both sides) that those stepping over the line probably have their own reasons (I'm not saying these reasons justify their behaviour, I'm simply saying they likely exist). Be it a real shitty day at the office, a life of oppression or anywhere in between. Far too often this consideration is beaten to the surface by their inner rage of injustice. How dare they presume I'm anything other than open and accepting of all gender, race, sexuality etc.

It's at this point Lee, Chris and Fran all have a choice. They can continue to be outraged and start to ridicule/attack each other, or they can simply state what needs to be clarified (I'm a girl / Sorry I called you a dude) and attempt to de-escalate from there on in.

It's THESE choices that need to be observed, it's THESE choices that a mediation team will seek to take action on and it's THESE choices that may see the revocation of karma (not using a gendered pronoun). You don't need to walk on eggshells, and you're allowed to slip up. What's important is how you (and the person you offended) react when you do.

So in the scenario @dshafik gave, there's absolutely nothing I personally would consider a breach of the proposed CoC. Something like this _SHOULD_ be an easy one to resolve. If Lee puts his hands up and apologises, and Chris/Fran continue to be "outraged" and try to pursue Lee by goading him into further dialogue. Then it may well be a CoC that helps prevent Fran/Chris from doing this. 

Another thing to note in this scenario is; if Fran did report this on behalf of Chris it may well be the case that a mediation team could help to engage them in reasoning and prevent further escalation. I'm not too sure how I feel about the team being involved at the "social media" level but it's certainly a possibility. Even if it's a call for a time-out, it'll be very apparent after that point who is/isn't looking to calm things down.

**Comment by TheFrozenFire** [comment]
I'd like to preface this by saying that regardless of my willingness to try to steer this proposal in a less dangerous direction, I'll continue to oppose any proposal that creates an enforcement process. That being said:

Frankly, I think that using controversial types of rudeness as examples of what we want to enforce against is a purely bad idea. Mis-gendering someone intentionally is absolutely _rude_. I don't think anyone reasonable would argue against that. Mis-gendering someone casually is a mistake, with no associated malice. If Fran or Chris can report this casual mis-gendering without being told to go away like the busybody they are, then this _would be_ "PC gone mad". In your scenario, "Lee" even apologized immediately after realizing his mistake. That should be the end of it. Anyone taking it further is just trying to create drama or further some personal vendetta. Such behaviours _should_ be punished, so that it's clear that this process has some integrity.

If you want anyone to take this CoC seriously, then focus on serious matters. Professional misconduct is pretty well-defined already. It applies to things like being punitive towards someone over a personal or political disagreement. It applies to being openly hostile towards someone (not just passionate). It applies to making defaming statements against someone (like accusing them of professional misconduct without evidence). It applies to discriminating against someone for attributes that are outside of their control.

In my opinion, it shouldn't even be considered professional misconduct to use derogatory terms, as long as you're not using them towards a colleague. Getting offended-by-proxy over anything is absolute madness. It's not injustice if there's no victim.

Focus on things that have actual discernible victims, where there's actual discernible harm. Not just discomfort.

**Comment by Swader** [comment]
>  Getting offended-by-proxy over anything is absolute madness. It's not injustice if there's no victim.
> 
> Focus on things that have actual discernible victims, where there's actual discernible harm. Not just discomfort.

That about sums it up, well done. Lee says "Whoops, sorry" and that should be the end of it in any sane society. The complaint of either Chris or Fran should not even be heard. 

If Chris complains, she should be told "you should probably identify by your full name, change your profile pic, or update your bio, just to avoid these misunderstandings in the future" and if Fran complains, they should be told "Are you serious? Please don't try to create drama - the reporting process is anonymous and private, and if someone has been offended please encourage them to come forth, rather than trying to proxy feelings".

**Comment by padraic** [comment]
> If Chris complains, she should be told

Definitely, do not do that. We really shouldn't be telling others what to do, and it's likely not to be received well.

I mostly agree with @dshafik's suggestion, though I would point out that being a "direct recipient" implies an individual. If you show up at a conference, unleash a dick pic, then who was the direct recipient? You can see where the circle starts...

Look, the COC is definitely not there to encourage campaigns of terror, of any description, on spurious grounds. The suggestion is actually rooted in what we do, or do not, state is "misconduct". Is getting a pronoun wrong a misconduct? What is the specific misconduct that would be claimed, as we currently list them? Can we redefine/clarify that specific misconduct to remove fuzziness and provide guidance to the COC team?

If it's a once off pronoun error, I certainly would not expect the COC to apply beyond offering the mediation process to ease over any misunderstanding.

**Comment by derickr** [comment]
> Rest assured, these people exist in great numbers, and the CoC gives them the power to keep complaining about someone until they find that one little tidbit that actually does offend random individual xyz and kick up a fuss around their private life. 

Perhaps, but that is then going to be thrown out quickly by the Community Mediation Team... especially if the reporter is not a party to the original issues (the Chris/Sam example from above for example). I'm not too worried about this one.

**Comment by TheFrozenFire** [comment]
> Look, the COC is definitely not there to encourage campaigns of terror, of any description, on spurious grounds. 

Perhaps not, but it certainly helps those who wish to enact them. You might be reasonable, but not everyone is.

> Perhaps, but that is then going to be thrown out quickly by the Community Mediation Team... especially if the reporter is not a party to the original issues (the Chris/Sam example from above for example). I'm not too worried about this one.

That assumes that the CMT isn't sympathetic to the complainer's complaining, or prejudiced against the accused. Both of those things have already occurred within the PHP community. You're completely ignoring human nature and assuming that _your friends_ are the ones who will be staffing the CMT, I think.
